src/features/user_profile/components/SettingsForm/SettingsForm.tsx

```diff
@@ -205,29 +186,24 @@ function SettingsFormGen({ userProfile, updateUserProfile }) {
                   />
                 </div>
 
-                {featureFlags?.[FeatureFlag.FEED_SELECTOR] && (
-                  <Select
-                    alwaysShowPlaceholder
+                {(featureFlags[FeatureFlag.FEED_SELECTOR] ||
+                  featureFlags[FeatureFlag.EVENTS_LIST__FEED_SELECTOR]) && (
```

Comment:
What's the difference between the two flags? shouldn't one of them be deprecated?

Reply:
to refactor in next iteration of feature configs